src/sonic-yang-models/tests/yang_model_tests/tests_config/sflow.json

```diff
@@ -0,0 +1,226 @@
+                        "alias": "eth0",
+                        "description": "Ethernet0",
+                        "lanes": "65",
+                        "mtu": 9000,
+                        "name": "Ethernet0",
+                        "speed": 25000
+                    }
+                ]
+            }
+        },
+        "sonic-sflow:sonic-sflow": {
+            "sonic-sflow:SFLOW_SESSION": {
+                "SFLOW_SESSION_LIST": [
+                    {
+                        "port": "Ethernet0",
+                        "admin_state": "up",
+                        "sample_rate": "40"
+                    }
+                ]
+            }
+        }
+    },
+
+    "SFLOW_SESSION_TEST_WITH_INCORRECT_PORT_TYPE_IN_SAMPLE_RATE": {
+        "sonic-sflow:sonic-sflow": {
+            "sonic-sflow:SFLOW_SESSION": {
+                "SFLOW_SESSION_LIST": [
+                    {
+                        "port": "all",
+                        "admin_state": "up",
+                        "sample_rate": "40000"
+                    }
+                ]
+            }
+        }
+    },
+
+    "SFLOW_TEST": {
+        "sonic-port:sonic-port": {
+            "sonic-port:PORT": {
+                "PORT_LIST": [
+                    {
+                        "admin_status": "up",
+                        "alias": "eth0",
+                        "description": "Ethernet0",
+                        "lanes": "65",
+                        "mtu": 9000,
+                        "name": "Ethernet0",
+                        "speed": 25000
+                    }
+                ]
+            }
+        },
+        "sonic-sflow:sonic-sflow": {
+            "sonic-sflow:SFLOW": {
+                "global": {
+                    "agent_id": "Ethernet0",
+                    "admin_state": "up",
+                    "polling_interval": "20"
+                }
+            }
+        }
+    },
+
+    "SFLOW_TEST_WITH_NON_EXIST_PORT": {
+        "sonic-sflow:sonic-sflow": {
+            "sonic-sflow:SFLOW": {
+                "global": {
+                    "agent_id": "Ethernet0",
+                    "admin_state": "up",
+                    "polling_interval": "20"
+                }
+            }
+        }
+    },
+
+    "SFLOW_TEST_WITH_INCORRECT_POLLING_INTERVAL": {
+        "sonic-port:sonic-port": {
+            "sonic-port:PORT": {
+                "PORT_LIST": [
+                    {
+                        "admin_status": "up",
+                        "alias": "eth0",
+                        "description": "Ethernet0",
+                        "lanes": "65",
+                        "mtu": 9000,
+                        "name": "Ethernet0",
+                        "speed": 25000
+                    }
+                ]
+            }
+        },
+        "sonic-sflow:sonic-sflow": {
+            "sonic-sflow:SFLOW": {
+                "global": {
+                    "agent_id": "Ethernet0",
```

Comment:
If agent ID is not mandatory for this test case we can remove it, keep shorter config.

Reply:
Addressed